soc/xtensa/intel_adsp/ace/power.c

```diff
@@ -407,6 +407,11 @@ void pm_state_exit_post_ops(enum pm_state state, uint8_t substate_id)
 	}
 
 	z_xt_ints_on(core_desc[cpu].intenable);
+
+	/* We don't have the key used to lock interruptions here.
+	 * Just set PS.INTLEVEL to 0.
+	 */
+	__asm__ volatile ("rsil a2, 0");
```

Comment:
This commit should be part of the previous one. Without restoring the `PS` value, `PS.INTLEVEL == 0`

Reply:
I can squash them.